src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java

```diff
@@ -163,11 +163,17 @@ protected FileMonitoringController(FilePath ws) throws IOException, InterruptedE
         // TODO would be more efficient to allow API to consolidate writeLog with exitStatus (save an RPC call)
         @Override public Integer exitStatus(FilePath workspace, Launcher launcher, TaskListener listener) throws IOException, InterruptedException {
             FilePath status = getResultFile(workspace);
-            if (status.exists()) {
+            if (status.exists() && status.length() > 0) {
```

Comment:
Well this alone would fix the issue, right?

Reply:
Theoretically, yes, if there's no bogus whitespace or other stuff in there.